cpu/stm32l0/include/periph_cpu.h

```diff
@@ -85,6 +85,17 @@ typedef struct {
     uint8_t ev_irqn;        /**< event IRQ */
 } i2c_conf_t;
 
+/**
+ * @brief   Define the number of available PM modes
+ */
+#define PM_NUM_MODES    (2U)
+
+/**
+ * @brief   Override the default initial PM blocker
+ * @todo   we block all modes per default, until PM is cleanly implemented
```

Comment:
i let the comment how it was specified for the other CPUs. But, the functions such as pm_block, pm_unblock seems to work on my side, so i don't know what is missing. (for example, at start up, i have made a pm_block(0) to prevent the CPU from going into standby mode)

But, for now, i'm still facing an issue, when my CPU is in stop mode, i have configured the RTC to trigger an alarm and wake up the CPU every 1 second, but the scheduler stop running the tasks and goes back immediatly in stop mode, moreover UART is printing wrong character at wake up. 
Am i missing something or doing something wrong ? Is it possible to run RIOT in stop mode ?